### PR TITLE
updates to the faq

### PR DESCRIPTION
**Pull request by sethladd**
Closes https://github.com/flutter/flutter/issues/4656

### CODE REVIEW

**Comment by sethladd** [comment]
cc @abarth

File: faq.md [hunk 1]
```diff
@@ -89,9 +90,8 @@ picture of the main components.
 ## How does Flutter run my code on Android?
 
 The engine's C/C++ code is compiled with Android's NDK, and the majority of the
-framework and application code is running on the Dart VM. The Dart VM generates
-JIT-compiled optimized native code on the device. (In other words, the Dart VM
-is not an interpreter.)
+framework and application code is running as native code,
+compiled by the Dart VM.
```
Comment (abarth) [inline faq.md]: Maybe say "running as native code compiled by the Dart compiler." ?

**Comment by abarth** [comment]
LGTM

File: faq.md (hunk 1 above)
Comment (sethladd) [inline faq.md]: sgtm. done!

**Pull request merged**